modules/programs/dircolors.nix

```diff
@@ -67,6 +67,19 @@ in {
         Extra lines added to <filename>.dir_colors</filename> file.
       '';
     };
+
+    configPath = mkOption {
```

Comment:
Could just remove this and unconditionally use `xdg.configFile`?

Reply:
The dir_colors man page does say

```
~/.dir_colors
       Per-user configuration file.
```

so in principle it is incorrect to place it anywhere else. But, I don't know of anything that uses this file except the line calling dircolors, which is under HM control. So I guess it would be OK to place it elsewhere.

In fact, it wouldn't be necessary to place it anywhere in the home directory, could simply reference it directly in the Nix store. I.e. something like

``` bash
eval $(${pkgs.coreutils}/bin/dircolors -b ${configFile})
```

where

```
configFile = pkgs.writeText "dir-colors" (concatStringsSep "\n" ([ ]
      ++ mapAttrsToList formatLine cfg.settings ++ [ "" ]
      ++ optional (cfg.extraConfig != "") cfg.extraConfig))
```